frontend/packages/ux-editor/src/components/Properties/EditSubformTableColumns/ColumnElement/EditColumnElement/EditColumnElementContent.tsx

```diff
@@ -40,12 +42,15 @@ export const EditColumnElementContent = ({
         />
       )}
 
-      <StudioDisplayTile
-        className={classes.componentCellContent}
-        label={t('ux_editor.properties_panel.subform_table_columns.column_cell_content')}
-        value={cellContent}
-        showPadlock={false}
-      />
+      {/* This is disable for now, and maybe will show it with new design for multiple data model binding  in a separate issue */}
```

Comment:
I think we should save and display the value of the cellContent within this PR, as they’re part of the same issue.

```suggestion
    <StudioDisplayTile
      className={classes.componentCellContent}
      label={t('ux_editor.properties_panel.subform_table_columns.column_cell_content')}
      value={cellContent}
      showPadlock={false}
    />
```

Reply:
Fixed 👍